setup.py

```diff
@@ -64,7 +64,7 @@ def read(*parts):
     install_requires=install_requirements,
     extras_require={
         'tests': test_requirements,
-        'filer': ['django-filer', ],
+        'filer': ['django-filer==1.2.4', ],
```

Comment:
I think we shouldn't pin an explicit version. Could we define a range or let it completely open?